src/util/AllocatorWithLimit.h

```diff
@@ -173,32 +177,37 @@ class AllocatorWithLimit {
     const auto bytesNeeded = n * sizeof(T);
     const bool wasEnoughLeft =
         memoryLeft_.ptr()->wlock()->decrease_if_enough_left_or_return_false(
-            bytesNeeded);
+            MemorySize::bytes(bytesNeeded));
```

Comment:
Directly define the `bytesNeeded` variable to be of type `MemorySize`

Reply:
Done, see [21022b3](https://github.com/ad-freiburg/qlever/pull/1112/commits/21022b3fedc90323b39a0a64ee34960059099dfc).